src/main/java/com/bp3x/raidbot/commands/lfg/util/Event.java

```diff
@@ -110,6 +110,16 @@ public enum EventPlayerStatus {
         TENTATIVE
     }
 
+    /**
+     * Return true if accepted or tentative participants exist on the event, false otherwise
+     * @return - boolean
+     */
+    public boolean getActiveParticipants() {
```

Comment:
naming nit: `hasActiveParticipants` would be in line w/  the "is" or "has" boolean convention